src/Raven.Server/Documents/Handlers/AI/Agents/ConversationDocument.cs

```diff
@@ -26,6 +26,7 @@ public class ConversationDocument([NotNull] string agent, BlittableJsonReaderObj
     public AiUsage TotalUsage = new AiUsage();
     public string ChangeVector;
     public string Id;
+    public int PreviousUsedTokens { get; set; }
```

Comment:
```suggestion
    public int PreviouslyUsedTokens { get; set; }
```

What does this mean? And why is that different from the `Usage` we already have?

Reply:
Removed it, and added AiUsage CurrentUsage instead.
But, it's different because `document.TotalUsage.PromptTokens` is just an accumulator. It doesn't represent the size of the previous message, it represents the sum of all prompt tokens from every turn added together.